src/pages/index.js

```diff
@@ -102,4 +104,19 @@ const IndexPage = () => {
     )
 }
 
+export const query = graphql`
+  query {
+    novelchan: file(relativePath: { regex: "/novel-chan/" }) 
+    {
+        childImageSharp {
+            fluid (
+                quality: 100
+            ) {
+                ...GatsbyImageSharpFluid
+            }
+        }
+    }
+  }
+`
+
```

Comment:
@TraceLD Here's the query I used...
It looks like you almost had it with what you had sent before, but the `eq: [filepath]` comparator wasn't cooperating so I regex'ed it based on your setup. We can change this per-picture or to do multiple pictures depending on our needs later, but this should work fine for now.

Also, this is _purely_ up to you, but I did not include the `maxWidth` / `maxHeight` properties here since the image itself is relatively small - due to this, I also bumped the quality up to 100% so that gatsby-image wouldn't hurt image quality either.